docs/radio.rst

```diff
@@ -132,6 +133,27 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_all()
```

Comment:
I vote -1 because it's too wordy. I believe `receive_all` - while not precise - communicates what's going on and allows kids with underdeveloped typing skills a bit of a break. ;-)